lib/filters/mongodb_version_filter.js

```diff
@@ -24,6 +24,8 @@ class MongoDBVersionFilter {
       return callback();
     }
 
+    var self = this;
```

Comment:
`const` ?

Reply:
there isn't an engine requirement, so I didn't want to use anything that required NODE 4+

Reply:
well you're modifying a `class` aren't you? :) Maybe throw in an engines requirement for node 4+